cli/js/runtime_main.ts

```diff
@@ -49,6 +49,11 @@ export function bootstrapMainRuntime(): void {
   }
   log("bootstrapMainRuntime");
   hasBootstrapped = true;
+  // Closes a denial of service vulnerability.  This makes Deno intentionally
+  // non-compliant with ECMA-262 Annex B.2.2.1.
+  // See: https://www.ecma-international.org/ecma-262/6.0/#sec-object.prototype.__proto__
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  delete (Object.prototype as any).__proto__;
```

Comment:
`bootstrapMainRuntime` is called when creating new isolate from snapshot; is there a way we could delete to prototype **before** snapshotting? I don't really know if that would make any difference, but potentially snapshot could be smaller?

Reply:
done... I don't think it makes a difference in size, but clearly a better place to put it.